### PR TITLE
Add 3× scaling setting

### PR DESCRIPTION
**Pull request by meisenzahl**
Related to https://github.com/elementary/switchboard-plug-display/issues/130

### CODE REVIEW

**Comment by meisenzahl** [comment]
@cassidyjames Could you maybe test this with LoDPI (e.g. Pinebook Pro) and HiDPI hardware?

**Review by cassidyjames — CHANGES_REQUESTED** [review]
On my 27" 2560×1440 display, this shows all three options (1x, 2x, and 3x scaling). 2x is debatable, but 3x makes no sense.

I wonder if we should restrict this to showing options for the equivalent of 768 vertical pixels, since that's our minimum supported resolution? So on a 27" 1440p display, we wouldn't show 2x or 3x because 1440/2 = 720, and 1440/3 = 480—far below our supported resolution.

File: src/Views/DisplaysView.vala [hunk 1]
```diff
@@ -44,9 +46,19 @@ public class Display.DisplaysView : Gtk.Grid {
...
             var dpi_label = new Gtk.Label (_("Scaling factor:"));
 
+            int width, height;
+            monitor_manager.virtual_monitors[0].get_current_mode_size (out width, out height);
+
             dpi_combo = new Gtk.ComboBoxText ();
-            dpi_combo.append_text (_("LoDPI"));
-            dpi_combo.append_text (_("Pixel Doubled"));
+            dpi_combo.append_text (_("LoDPI") + " (1×)");
+            int max_scale = height / 768;
```
Comment (cassidyjames) [inline src/Views/DisplaysView.vala:54]: Should probably pull this out into a constant so it's more obvious where it's coming from and what it represents, e.g. `MIN_SUPPORTED_HEIGHT`

**Review by cassidyjames — COMMENTED** [review]
Hm, now when I use this on a HiDPI laptop with a loDPI external display, the scaling factor option is disabled completely (forced to 1×). I wonder if we should just make sure _at least one_ display or maybe the _primary_ display meets the 768 minimum height when considering the scaling factors? 

I'm not sure how this would behave without having a previous display arrangement (I previously manually set my laptop resolution to half and the scaling-factor to 1×), but it would not be fun to accidentally get into a case where your HiDPI laptop was forced to LoDPI because you connected an external display, without it also halving the resolution...

Gah multi-monitor and multi-scaling-factor is so complex to handle all the edge cases :sob:

**Comment by meisenzahl** [comment]
@cassidyjames this is really dumb now :grin:

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**